temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -525,3 +528,24 @@ message DeleteServiceAccountResponse {
     // The async operation.
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
+
+message GetAuditLogsRequest {
```

Comment:
Is the service entry missing for this req/resp RPC?

Reply:
Yup just added